sdk/zksync.js/README.md

```diff
@@ -1,15 +1,16 @@
-# JavaScript SDK for zkSync
+# JavaScript SDK for RIF Aggregation
 
-**zkSync** is a scaling and privacy engine for Ethereum. Its current functionality scope includes low gas transfers of
-ETH and ERC20 tokens in the Ethereum network.
+**RIF Aggregation** is a scaling and privacy engine for RSK. Its current functionality scope includes low gas transfers
+of RBTC and ERC20 tokens in the RSK network.
 
-This document is a description of the JS library that can be used to interact with zkSync.
+This document is a description of the JS library that can be used to interact with RIF Aggregation.
 
 ## Usage
 
-Full reference on how to use the library can be found [here](https://zksync.io/api/sdk/js/).
+Full reference on how to use the library can be found [here](https://developers.rsk.co/rif/aggregation/).
```

Comment:
For the SDK reference, I'll still point to their site, since we don't have anything related to the SDK in the dev portal

Reply:
Done